spawn-host.patch

```diff
@@ -0,0 +1,21 @@
+--- ~/src_assets/linux/assets/apps.json
++++ ~/src_assets/linux/assets/apps.json
+@@ -12,15 +12,15 @@
+       "image-path": "desktop.png",
+       "prep-cmd": [
+         {
+-          "do": "xrandr --output HDMI-1 --mode 1920x1080",
+-          "undo": "xrandr --output HDMI-1 --mode 1920x1200"
++          "do": "flatpak-spawn --host xrandr --output HDMI-1 --mode 1920x1080",
++          "undo": "flatpak-spawn --host xrandr --output HDMI-1 --mode 1920x1200"
```

Comment:
instead you should ship xrandr. But how does this work with wayland?

Reply:
The user has several options to choose (Wayland compatible) this one is X11 specific. This part of the patch is no longer needed by including xrandr with build